src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -28,7 +28,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        logger.info("Data from DB are successfully fetched. Param: userId={}", userId);
+        logger.debug("Data from DB are successfully fetched. Param: userId={}", userId);
```

Comment:
this log can be kept as info, since it has information related to business logic